script/delete-harbor.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Constants
+ROOT_DIR="$(cd "$( dirname "${BASH_SOURCE[0]}" )/.." >/dev/null && pwd)"
+
+# Load Libraries
+# shellcheck disable=SC1090
+. "${ROOT_DIR}/script/libtest.sh"
+# shellcheck disable=SC1090
+. "${ROOT_DIR}/script/liblog.sh"
+
+namespace="harbor"
+while [[ "$#" -gt 0 ]]; do
+    case "$1" in
+        -n|--namespace)
+            shift
+            namespace="${1:?missing namespace}"
+            ;;
+        *)
+            echo "Invalid command line flag $1" >&2
+            return 1
+            ;;
+    esac
+    shift
+done
+
+# Uninstall Harbor
+silence kubectl delete ns "$namespace"
```

Comment:
I'm guessing you know this works, but wouldn't this be a race condition, or leave helm dangling, or make the next line redundant? That is, shouldn't we uninstall harbor first. Ah, it looks like you've repeated the namespace delete below afterwards, so I'm guessing you meant to remove this one?

Reply:
I introduced this line twice by mistake. I'll fix it.